man/dnf.plugin.kpatch.8

```diff
@@ -0,0 +1,34 @@
+.TH man 8 "23 Apr 2020" "1.0" "dnf-kpatch man page"
```

Comment:
I'd rather leave out `extra3` argument of `.TH` macro, as it is substituted by the section description.